files/ru/web/css/vertical-align/index.md

```diff
@@ -231,6 +253,7 @@ img.middle {
 
 ## Смотрите также
 
+- [Typical use cases of Flexbox, section "Center item"](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Typical_use_cases_of_flexbox#center_item)
```

Comment:
Russian version of https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Typical_use_cases_of_flexbox not created yet. So `/en-US/docs/Web/CSS/CSS_flexible_box_layout/Typical_use_cases_of_flexbox#center_item` is valid link.

Reply:
Perhaps this is erroneous behavior https://github.com/mdn/translated-content/blob/main/scripts/check-url-locale.js :thinking:

@yin1999 what do think? Need your help :slightly_smiling_face:

Reply:
Yes, changing the locale of the link to `ru` will make the link invalid (the users could not go the the corrected section). But, I think it would still be nice to "tell" the readers that the pointed article is not localized (which means, if the article is important, we should raise an issue or create a PR to translate it as soon as possible), like this one:

![image](https://github.com/mdn/translated-content/assets/15844309/07139fa5-6c87-4761-b526-0a4b89c50396)

If the reader click this link and find the article is not translated, many of them may not want to continue reading.

Reply:
@yin1999 thank yoy for reply!

Unfortunately, the translation will take a lot of time :slightly_frowning_face:. More precisely, not even the translation itself, but the translation and review.

I think that there should be notification of such problems, but this should not be an obstacle to the merging PR.

And your option is very good when the user is shown a notification that the page is currently available only in English. As far as I understand, for this you need to make changes to the yari?

Thus, at the moment I have two options?
1. remove the link
2. change it to `ru` (that is, it will link to a non-existent page)

And both are bad =) But I think it is better to remove link)